storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc

```diff
@@ -14,7 +14,7 @@ Familiarity with xref:../../storage/understanding-persistent-storage.adoc#unders
 
 To create CSI-provisioned PVs that mount to AliCloud Disk storage assets, {product-title} installs the AliCloud Disk CSI Driver Operator and the AliCloud Disk CSI driver, by default, in the `openshift-cluster-csi-drivers` namespace.
 
-* The _AliCloud Disk CSI Driver Operator_ provides a storage class (`alicloud-disk`) that you can use to create persistent volume claims (PVCs). The AliCloud Disk CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on demand, eliminating the need for cluster administrators to pre-provision storage.
+* The _AliCloud Disk CSI Driver Operator_ provides a storage class (`alicloud-disk`) that you can use to create persistent volume claims (PVCs). The AliCloud Disk CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on demand, eliminating the need for cluster administrators to pre-provision storage. You can disable this default storage class if desired (see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc[Managing the default storage class]).
```

Comment:
s/storage class if desired/storage class. if desired

Reply:
I don't think that this is supposed to be two sentences, so I let it be.

Reply:
@lpettyjo  That was supposed to be a comma, not a period. Sorry!

>  You can disable this default storage class, if desired